src/main/java/uk/gov/cshr/domain/Role.java

```diff
@@ -6,15 +6,16 @@
 public class Role {
     @Id
     @GeneratedValue(strategy = GenerationType.AUTO)
-    private Long id;
+    private long id;
 
     @Column(length = 100, unique = true)
     private String name;
 
     @Column
     private String description;
 
-    protected Role() {
+    public Role() {
```

Comment:
Is it valid to have a role object with no name? If this constructor is here to make Spring work, change it to `protected`